src/collections/_documentation/workflow/user-settings.md

```diff
@@ -0,0 +1,187 @@
+
+{% asset user-settings/us-avatar.png alt="Avatar field in Account Details under User Settings" width="227.5" %}
+
+
+# Security
+
+Security contains two sections: Settings and Session History.
+
+## Settings
+### Password
+
+We recommend changing your password frequently. However, keep in mind that changing your password will invalidate all logged in sessions.
+
+{% asset user-settings/us-pass.png alt="Password field in Security under User Settings" width="450" %}
+
+### Sessions
+
+Sessions end with each sign out. Clicking "Sing out of all devices" will end your sessions with all Sentry devices.
+
+{% asset user-settings/us-sess.png alt="Sessions field in Security under User Settings" width="550" %}
+
+### Two-Factor Authentication
+
+Keep your account information secure by enabling a two-factor authentication method. Choose between an Authenticator App, Text Message, or a Universal 2nd Factor (U2F) hardware device.
+
+{% asset user-settings/us-2fact.png alt="Two-Factor Authentication selection in Securty under User Settings" width="745" %}
+
+### Recover Codes
+
+Use recovery codes to access your account if you lose your device and cannot receive two-factor authentication codes. Click the "View Codes" button to download, print, or copy your codes to a secure location.
+
+{% asset user-settings/us-rec.png alt="Recover codes field in Security under User Settings" width="673" %}
+
+### Session History
+
+Find a historical list of sessions, including First Seen and Last Seen timestamps, under the Session History tab.
+
+{% asset user-settings/us-shistory.png alt="Session History tab in Security under User Settings" width="380" %}
+
+# Notifications
+
+Sentry notifications are broadly categorized into alerts and non-alert notifications. Alerts can be sent to many supported [integrations]({%- link _documentation/workflow/integrations/index.md -%}#webhook-alerts). Non-alert notifications only go to email.
+
+## Alerts & Notifications
+
+You can define Project Alerts in  [Project] > Project Settings > Alerts. Slide the toggle under Send Me Alerts to receive team alerts. You can also fine-tune the alerts you receive per project.
+
+Control workflow notifications (for example, in issue assignment, resolution status, and comments) by clicking a radio button under Send Me Workflow Notifications. You can also fine-tune workflow notifications per project.
+
+For more details, see the [full documentation on Alerts & Notifications]({%- link _documentation/workflow/alerts-notifications/index.md -%})
+
+## Email Routing
+
+Route email notifications to an alternative email address and keep your project notifications organized. 
+
+{% asset user-settings/us-eroute.png alt="Email Routing field in Notifications under User Settings" width="450" %}
+
+## Weekly Reports
+
+Sentry generates reports per project and sends them once a week. You can fine-tune your reports per project.
```

Comment:
Is there a day it's sent? Like, each Friday, or every seven days after initial set up?

Reply:
Checking on this.